executor/src/lib.rs

```diff
@@ -448,6 +449,8 @@ where
         // This must be done before calculate potential skipping of transactions in idempotent commit.
         let mut txns_to_keep = vec![];
         let mut blocks_txns = vec![];
+        let mut reconfig_events = vec![];
+        let reconfig_event_key = ValidatorSet::change_event_key();
```

Comment:
you probably know I don't like inconsistency, use the same name and maybe extract a common function